Updates to the Annual Report Process and Template for LFDT

swcurran opened the pull request:
Signed-off-by: Stephen Curran <swcurran@gmail.com>

tkuhrt committed:
update for LFDT

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
prepare for LFDT

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
update for LFDT; includes two FIXMEs

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Update for LFDT

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
update for LFDT (TOC->TAC)

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Revert TOC-TAC changes

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Update for LFDT; added OpenSSF Scorecard and FIXMEs sections

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Update for LFDT

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Fix broken link

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

mbrandenburger committed:
Update project-updates

- revert changes on history to no break working links
- add TODOs to mark changes that needs to be made once information is
  available

Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

mbrandenburger committed:
Use github.com/LF-Decentralized-Trust

Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

tkuhrt committed:
Update navigation order

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

tkuhrt committed:
Additional FIXMEs

Signed-off-by: Tracy A. Kuhrt <tracy.a.kuhrt@accenture.com>

swcurran committed:
Updates to the Annual Report Process and Template for LFDT

Signed-off-by: Stephen Curran <swcurran@gmail.com>

swcurran commented:
Changes in the PR:

- Hyperledger references to LFDT
- lifecycle "stage" references --> "status"
- Change references to Maintainers to TSC
- Added "TBD Link" for links that are TBD (e.g., Discord channel)
- Fixed Markdown lint warnings
- General text cleanup

arsulegai commented on project-reports/0000-annual-review-template.md:
```diff
@@ -6,36 +6,43 @@ grand_parent: LF Decentralized Trust TAC
 
-# Project Health
-<mark>_Include a link to your project’s [LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/projects). We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add color to the numbers and graphs we will see on Insights._
+## Project Health
```
[nitpick] I’m curious why the entire file appears as different. It seems like an extra # was added for the headers.

swcurran replied:
That was cleanup of the headers — fixing Markdown Lint warnings.  The front matter puts in the “#” header, so the rest should be “##” headers.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
+- Agreeing between themselves who will complete the annual review.
+- Informing the community about the annual review and requesting input.
+- Creating the annual review by filing a PR in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
+- Sending email to the [TAC mailing list](mailto:toc@lists.lf-decentralized-trust.org) and to the project community so that the community knows the PR is there and can comment on it.
```
Q: Is the mailing list a TODO?

swcurran replied:
Definitely need a `toc` to `tac`, but yes, it is a ToDo.

swcurran replied:
Just made the `tac` to `toc` change, but I think a link checker can find this one if it is wrong.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
+
+- Agreeing between themselves who will complete the annual review.
+- Informing the community about the annual review and requesting input.
+- Creating the annual review by filing a PR in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```
Q: Is the github link a TODO?

swcurran replied:
I think I just need to change `toc` to `tac` in it.  But a link check should be run on all of these docs after publication to make sure they are right.

swcurran replied:
Changed the `toc` to `tac` where appropriate. There is one instance that was purposely left.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
 ### TAC Secondary Member
-- Working closely with the lead TAC member to review and corroborate their findings
+
+- Working closely with the lead TAC member to review thew project's annual report, and corroborate and publish their findings.
```
```suggestion
- Working closely with the lead TAC member to review thew project's annual report, and to corroborate and publish their findings.
```

swcurran replied:
Fixed

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-* A majority of the TAC members agree to continue the project at its current stage, or
-* A majority of the TAC members recommend that a project be moved to a new stage, including `End of Life`, or
-* If the TAC cannot come to a consensus, the project will remain at its current stage.
+- A majority of the TAC members agree to continue the project at its current status, or
```
Q: Is the term `project's status` broader than `project's stage` or do they carry the same meaning? If `status` is broader, does it imply an inferred condition based on the project's report and current stage?

swcurran replied:
As per the discussion at the TOC call on 20240829 — the term “status” and “stage” were used seemingly interchangeably in the document.  I didn’t think that was clear, so I checked the “lifecycle” document and it uses the term “status”, so I changed all “stage” references to “status”.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-        - remove the `nav_exclude` line
-    - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
-* Send an email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+Project TSC members are responsible for agreeing between them who will complete the annual review. One of the TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```
```suggestion
Project TSC members are responsible for agreeing between them who will complete the annual review. One of the project's TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```

swcurran replied:
Fixed

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-        - remove the `nav_exclude` line
-    - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
-* Send an email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+Project TSC members are responsible for agreeing between them who will complete the annual review. One of the TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```
Q: Should the github link be a TODO?

swcurran replied:
The links can be checked by a link checker when the new repo is used.  I don’t think we need a todo on each where we probably have it right.  I did put a couple of “TBDs” where I didn’t know at all what the link would be.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-* A TAC representative will be chosen on a round-robin basis to lead the review once the project files a PR. This representative should be someone that is not directly involved with the project.
-* A secondary representative will be chosen on a round-robin basis to observe and corroborate the findings of the lead. This representative should be someone that is not directly involved with the project.
-* The lead TAC member reviews the content of the PR and analyzes the project for community health indicators, their findings are placed within a thread in the TAC channel ([#toc](https://discord.com/channels/905194001349627914/941384040316018790)) for discussion. The thread should contain:
-    1. important facts about the project that could influence the TACs decision around the future of the project, its current stage, and path to other stages.
```
I notice that these sentences have been removed. It seems necessary to establish guidance that suggests the review should be objective and holistic, considering all aspects such as encouraging projects to plan for the next stage in their lifecycle, as well as ensuring ongoing viability.

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-Annual reviews will replace the Q1 quarterly report.  **If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
 
-**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
+Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
```
```suggestion
Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, the TAC will take this as a sign that the project is not under active maintenance and is likely to decide to archive the project and move it to End of Life status.**
```

swcurran replied:
Fixed

arsulegai commented on governing-documents/project-annual-review.md:
```diff
@@ -10,81 +10,89 @@ nav_order: 5
-**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
+Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled the project's TSC can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
```
Q: Would project's TSC decide to EOL their project or do they bring it to LFDT TAC for the move?

swcurran replied:
I believe the TSC has the governance authority to do it. The mechanics would be situational. This sentence is a cleanup / clarification (at least it clarified it for me) on what was in the document already.

swcurran commented:
I obviously have way too many commits in this — branched from the wrong fork.  Will close this, open a new PR and reference this  PR to capture the review comments made.  I’ve addressed all of the comments from the review.
